tests/test_security_jwt_general_optional.py

```diff
@@ -196,27 +188,29 @@ def test_security_jwt_access_token_changed():
     assert response.json() == {"msg": "Create an account first"}
 
 
-def test_security_jwt_access_token_expiration(mocker: MockerFixture):
+@pytest.mark.parametrize("jwt_backend", [AuthlibJWTBackend, PythonJoseJWTBackend])
+def test_security_jwt_access_token_expiration(mocker: MockerFixture, jwt_backend):
+    client, _ = create_example_client(jwt_backend)
     access_token = client.post("/auth").json()["access_token"]
 
-    mocker.patch("jose.jwt.datetime", _FakeDateTimeShort)  # 3 min left
-
+    mock_now_for_backend(mocker, jwt_backend, minutes=3)  # 3 min left
     response = client.get(
         "/users/me", headers={"Authorization": f"Bearer {access_token}"}
     )
     assert response.status_code == 200, response.text
     assert response.json() == {"username": "username", "role": "user"}
 
-    mocker.patch("jose.jwt.datetime", _FakeDateTimeLong)  # 42 days left
-
+    mock_now_for_backend(mocker, jwt_backend, days=42)  # 42 days left
```

Comment:
all the tests changes are about:
* make it parametrize to test the two backends
* mock datetime.now or time.time depending of the backend used (if necessary)
* create a client depending of the backend used (need to be dynamic and wrapped inside a `create_example_client` to really test the backends)